pkg/daemon/ceph/osd/volume.go

```diff
@@ -604,6 +606,12 @@ func getCephVolumeLVMOSDs(context *clusterd.Context, clusterName string, cephfsi
 			if osd.Type == "journal" {
 				store = "filestore"
 			}
+
+			// If no lv is specified let's take the one we discovered
+			if lv == "" {
```

Comment:
when would `lv` be the empty string? Doesn't line 580 require it to be something? Or it will simply list all osds if it's empty?

Reply:
Yes, it lists everything if empty, it will be empty in this scenario.